build/configs/imxrt1020-evk/scripts/memory.ld

```diff
@@ -85,17 +85,25 @@
 
 MEMORY
 {
-  /* 64Mb of HyperFLASH */
+  /* 8MB of HyperFLASH */
 
-  kflash (rx)  : ORIGIN = 0x60000000, LENGTH = 1M
-  uflash (rx)  : ORIGIN = 0x60200000, LENGTH = 1M
+  kflash (rx)  : ORIGIN = 0x60000000, LENGTH = 256K
+  uflash (rx)  : ORIGIN = 0x60040000, LENGTH = 256K
   flash  (rx)  : ORIGIN = 0x60400000, LENGTH = 62M
```

Comment:
Don't you need to modify LENGTH of flash?

Reply:
Actually, that value is never used. So I have not updated it. 
Also, if we need to specify the length, then it has to be the remaining length
which is 8M - 512K = 7.5M, 
and I don't know how to specify this value in the ld script. 

I think its ok to keep it for now. This line will be removed when we move to the auto-generate method for generating ld script from config file.